caluma/form/models.py

```diff
@@ -31,7 +31,7 @@ class FormQuestion(UUIDModel):
     sort = models.PositiveIntegerField(editable=False, db_index=True, default=0)
 
     class Meta:
-        ordering = ("-sort", "id")
+        ordering = ("-sort",)
```

Comment:
I'd still leave the "id" (or alternatively, "name") as secondary ordering field, so it remains deterministic even if the user/frontend gives the same "sort" value for multiple rows

Reply:
this is hidden from the user interface. User interface resp. api user doesn't see the sort and this is only handled internally where we make sure those numbers are never the same.

Reply:
Aah. If it's not even exposed via API, then I'm fine with it, as we can enforce consistency